server/session/handler_enchanting.go

```diff
@@ -0,0 +1,341 @@
+package session
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/block"
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/internal/sliceutil"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+	"github.com/sandertv/gophertunnel/minecraft/protocol/packet"
+	"golang.org/x/exp/slices"
+	"math"
+	"math/rand"
+)
+
+// enchantNames are names translated to the 'Standard Galactic Alphabet' client-side. The names generally have no meaning
+// on the vanilla server implementation, so we can sneak some easter eggs in here without anyone noticing.
+var enchantNames = []string{
+	"dragonfly",
+	"gophertunnel",
+	"go raknet",
+	"go lang",
+	"sandertv",
+	"t 14 raptor",
+	"da pig guy",
+	"potatoe train yt",
+}
+
+const (
+	// enchantingInputSlot is the slot index of the input item in the enchanting table.
+	enchantingInputSlot = 0x0E
+	// enchantingLapisSlot is the slot index of the lapis in the enchanting table.
+	enchantingLapisSlot = 0x0F
```

Comment:
Would use lowercase hex for consistency.

Reply:
Renamed.